docs/core/diagnostics/dotnet-stack.md

```diff
@@ -0,0 +1,116 @@
+---
+title: dotnet-stack tool - .NET Core
+description: Installing and using the dotnet-stack command-line tool.
+ms.date: 11/21/2019
+---
+# dotnet-stack performance analysis utility
+
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
+
+## Install dotnet-stack
+
+### Option 1 - Install as dotnet global tool:
+
+Install `dotnet-stack` [NuGet package](https://www.nuget.org/packages/dotnet-stack) with the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+
+```dotnetcli
+dotnet tool install --global dotnet-stack
+```
+
+### Option 2 - Direct download:
+
+- Win ([x86](https://aka.ms/dotnet-stack/win-x86) | [x64](https://aka.ms/dotnet-stack/win-x64) | [arm](https://aka.ms/dotnet-stack/win-arm) | [arm-x64](https://aka.ms/dotnet-stack/win-arm64))
+- macOS ([x64](https://aka.ms/dotnet-stack/osx-x64))
+- Linux ([x64](https://aka.ms/dotnet-stack/linux-x64) | [arm](https://aka.ms/dotnet-stack/linux-arm) | [arm64](https://aka.ms/dotnet-stack/linux-arm64) | [musl-x64](https://aka.ms/dotnet-stack/linux-musl-x64) | [musl-arm64](https://aka.ms/dotnet-stack/linux-musl-arm64))
+
+## Synopsis
+
+```console
+dotnet-stack [-h, --help] [--version] <command>
+```
+
+## Description
+
+The `dotnet-stack` tool:
+
+* Is a cross-platform .NET Core tool.
+* Reports the managed stacks for all threads in .NET process.
```

Comment:
What do you think about changing this to `Captures and writes out the managed stacks for all threads in the target .NET process?`

Reply:
That works too.  I used report here since I copied this over from the dotnet-trace docs which seemed to try to match the phrasing with the command verb, e.g., collect for `dotnet-trace collect`.

Reply:
Interesting. Some tools have more than one commands (i.e. counters, trace both have/will have collect/monitor) so it might not make the most sense to use the commands to describe the tools for those two tools.

Do you anticipate `dotnet-stack` to have more than just `report` command anytime soon?

Reply:
I don't anticipate `dotnet-stack` to have more verbs than `report`.  I already changed it to your suggestion locally.  I just need to push it up.